patches/tModLoader/Terraria.ID/GlowMaskID.cs.patch

```diff
@@ -0,0 +1,16 @@
+--- src/Terraria\Terraria.ID\GlowMaskID.cs
++++ src/tModLoader\Terraria.ID\GlowMaskID.cs
+@@ -1,3 +_,4 @@
++using ReLogic.Reflection;
+ using System;
+ 
+ namespace Terraria.ID
+@@ -258,5 +_,7 @@
+ 		public const short ArkhalisPants_Female = 250;
+ 		public const short ArkhalisWings = 251;
+ 		public const short Count = 252;
++
++		//public static readonly IdDictionary Search = IdDictionary.Create<GlowMaskID, short>();	Can't use with `static` classes!
```

Comment:
Is there a reason why we wouldn't want these functionalities in the commented classes?

Reply:
They're `static` classes, and create an error. Changing them to non-`static` classes would throw exceptions with mods that use them. This is a change for a hypothetical future tModLoader major versioned release.

Reply:
Ah, copy that. In that case I recommend none of these features are part of the ID classes, since that'd be incompatible for these. In that case they should be present in a new custom class from tModLoader.

Reply:
Is there a way to get an `IdDictionary` (or equivalent) with a `static` class? Otherwise, it won't matter to make a custom class as a layer on top of these. Keep in mind their only use is to map the `public const short` field names to their `type`s in a dictionary data structure; maybe other means are available?